configure-ironic.sh

```diff
@@ -43,5 +43,9 @@ endpoint_override = http://${IRONIC_URL_HOST}:5050
 endpoint_override = http://${IRONIC_URL_HOST}:6385
 EOF
 
+# oslo.config also supports Config Opts From Environment, log them
+echo '# Options set from Environment variables' | tee /etc/ironic/ironic.conf
+env | grep "^OS_" | tee -a /etc/ironic/ironic.conf
```

Comment:
This is adding shell environment variables directly to the config file. Could that cause parse issues? Should they be added as comments?

Reply:
thanks, that was a typo, I meant to add them to a different file.

Reply:
Ah, cool, I assumed you wanted everything in one place for debugging. A separate file seems even safer, though.